OoyalaSkinSampleApp/app/src/main/java/com/ooyala/sample/players/AudioOnlyPlayerActivity.java

```diff
@@ -0,0 +1,64 @@
+package com.ooyala.sample.players;
+
+import android.os.Bundle;
+import android.util.Log;
+import com.ooyala.android.OoyalaPlayer;
+import com.ooyala.android.PlayerDomain;
+import com.ooyala.android.configuration.Options;
+import com.ooyala.android.skin.OoyalaSkinLayoutController;
+import com.ooyala.android.skin.configuration.SkinOptions;
+import com.ooyala.sample.R;
+import com.ooyala.sample.utils.AudioOnlyPlayerInfo;
+import org.json.JSONObject;
+
+public class AudioOnlyPlayerActivity extends AbstractHookActivity {
+
+  @Override
+  public void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.player_skin_simple_layout);
```

Comment:
I think it's better to setContentView in `AbstractHookActivity`
Create there 
![image](https://user-images.githubusercontent.com/43542852/48046721-819a6480-e1a6-11e8-8732-3520c5a36a8b.png)
Override in deferred classes, and in AbstractHookActivity in onCreate call setContentView(getLayoutRes())

Reply:
Lets rewrite the whole sample-apps project in a separate pull request. This is just an audio only sample, not a refactoring.

Reply:
Ok